apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ProposalOverview/ProposalOverview.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ButtonStyle,
+  FormGroup,
+  Icon,
+  IconNames,
+  Input,
+  InputSize,
+} from '@sovryn/ui';
+
+import { translations } from '../../../../../../../locales/i18n';
+import {
+  ProposalOverviewData,
+  ProposalType,
+} from '../../NewProposalForm.types';
+
+export type ProposalOverviewProps = {
+  value: ProposalOverviewData;
+  onChange: (value: ProposalOverviewData) => void;
+  proposalType: ProposalType;
+  onBack: () => void;
+};
+
+export const ProposalOverview: FC<ProposalOverviewProps> = ({
```

Comment:
Why is this called `ProposalOverview`? I think `ProposalDataForm` is more suitable. Adjust all the related tabs and enums.